Makefile

```diff
@@ -231,6 +231,10 @@ endif
 
 EXTRA_DOCKER_ARGS += -v $(GOMOD_CACHE):/go/pkg/mod:rw
 
+ifdef LOCAL_BUILD
+EXTRA_DOCKER_ARGS+=-v $(CURDIR)/../libcalico-go:/go/src/github.com/projectcalico/libcalico-go:rw
```

Comment:
Does this actually need write permissions? That feels kinda scary to me - will go modules overwrite your local libcalico checkout?

Reply:
@rafaelvanoni

Reply:
@rafaelvanoni Do you know the answer to @caseydavenport 's question?  (My change here just moves the existing mount a bit earlier in the Makefile, so I haven't originated the `:rw`.)